types.proto

```diff
@@ -52,6 +54,10 @@ message KeyMetadata {
 	// CasRequired specifies if the cas parameter is 
 	// required for this key
 	bool cas_required = 8;
+
+	// VersionTtl specifies how long to keep versions around. If empty
+	// value, defaults to the configured version_ttl for the mount.
+	google.protobuf.Duration version_ttl = 9;
```

Comment:
After talking with @briankassouf, I will be renaming this to `version_TTL` (same as above)